### PR TITLE
refactors choronological test to use find(nth element)

### LINKED ISSUE #42: Refactor test for ordering posts on page
Can use expect(page.find('li:nth-child(1)')).to have_content 'This'

Create <ul> and make each new post a <li> element

### PR DESCRIPTION
**Pull request by runticle**
fixes #42

### CODE REVIEW

File: app/views/posts/index.html.erb [hunk 1]
```diff
@@ -7,15 +7,17 @@ Welcome to Acebook <%= current_user.name %>
...
-  <p><%= post.updated_at.strftime("%eth %b %Y %T") %></p>
-  <% if post.user == current_user %>
-    <%= link_to '❌', post_path(post), method: :delete, id: "delete-#{post.id}" %>
-    <% if post.editable? %>
-      <%= link_to '✏️', edit_post_path(post.id) %>
+  <li>
+    <%= simple_format(post.message) %>
+    <p><%= post.user.name %></p>
+    <p><%= post.updated_at.strftime("%eth %b %Y %T") %></p>
+    <% if post.user == current_user %>
```
Comment (runticle) [inline app/views/posts/index.html.erb]: it's very ugly

Comment (runticle) [reply app/views/posts/index.html.erb]: but who cares?

Comment (runticle) [reply app/views/posts/index.html.erb]: we can refactor it later when all is said and done

**Comment by rsmith88** [comment]
Let us know if you want further refactoring...

File: spec/features/posts_reverse_chrono_spec.rb [hunk 2]
```diff
@@ -15,6 +15,6 @@
     click_button 'Submit'
     time_2 = Time.now
 
-    expect(page).to have_content("My second post! Alfie #{time_2.strftime('%eth %b %Y %T')} ❌ ✏️  My first post! Alfie #{time_1.strftime('%eth %b %Y %T')} ❌ ✏️ ")
+    expect(page.find('li:nth-child(1)')).to have_content("My second post!")
```
Comment (runticle) [inline spec/features/posts_reverse_chrono_spec.rb]: a brilliant idea from @imogenmisso !

Comment (runticle) [reply spec/features/posts_reverse_chrono_spec.rb]: works a treat!

File: app/views/posts/index.html.erb (hunk 1 above)
Comment (imogenmisso) [reply app/views/posts/index.html.erb]: Ok

File: spec/features/posts_reverse_chrono_spec.rb (hunk 2 above)
Comment (toddpla) [reply spec/features/posts_reverse_chrono_spec.rb]: great addition nice one @imogenmisso

Comment (imogenmisso) [reply spec/features/posts_reverse_chrono_spec.rb]: Yay! Nice work @rsmith88 and Team Member 2 for implementing it!

Comment (imogenmisso) [inline spec/features/posts_reverse_chrono_spec.rb]: 🌟🌟

**Pull request merged**